README.md

```diff
@@ -214,10 +216,12 @@ A curated list of awesome resources, learning materials, tools, frameworks, plat
 ### Tutorials
 
   * [A Tutorial Implementation of a Dependently Typed Lambda Calculus](https://www.andres-loeh.de/LambdaPi/).
+  * [Algorithm W Step By Step](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.65.7733&rep=rep1&type=pdf).
```

Comment:
It's a tutorial that takes a paper format; it doesn't present any research. Therefore I believe it's a tutorial.